measure.rb

```diff
@@ -3535,14 +3539,10 @@ def self.add_component_loads_output(runner, model)
     objects_already_processed = []
 
     # EMS Sensors: Global
-
-    liv_load_sensors = {}
-
-    liv_load_sensors[:htg] = OpenStudio::Model::EnergyManagementSystemSensor.new(model, "Heating:EnergyTransfer:Zone:#{@living_zone.name.to_s.upcase}")
-    liv_load_sensors[:htg].setName("htg_load_liv")
-
-    liv_load_sensors[:clg] = OpenStudio::Model::EnergyManagementSystemSensor.new(model, "Cooling:EnergyTransfer:Zone:#{@living_zone.name.to_s.upcase}")
-    liv_load_sensors[:clg].setName("clg_load_liv")
+    # Need to resolve conflict after new component load approach is merged
+    liv_load_sensors = OpenStudio::Model::EnergyManagementSystemSensor.new(model, "Zone Predicted Sensible Load to Setpoint Heat Transfer Rate")
```

Comment:
It looks like using the new adjusted loads reported in EMS is more proper for dehumidifier, will test switching.

Reply:
Switching to adjusted loads made discrepancy smaller. Switched for dehumidifier, and reverted these lines for other systems.